src/models/validators/SpecialistTestsCommonSchemaCancelled.ts

```diff
@@ -40,6 +41,7 @@ export const testTypesCommonSchemaSpecialistTestsCancelled =
     defects: Joi.array()
```

Comment:
is ivaDefects replacing defects - should this be removed?

Reply:
I don't think it should be, as the logic before was expecting it to always be there regardless of the test type just not populated in the case of specialist tests